pkg/packagekit/updates.jsx

```diff
@@ -397,6 +430,208 @@ const UpdatesList = ({ updates }) => {
     );
 };
 
+// TODO replace by a shared component from dashboard page when dashboard redesigned is ready
```

Comment:
Did you mean "Overview"? Indeed @marusak is working on that in PR #15139. Feel free to include these commits in your PR, this is going to take a while still anyway.

By all means let's share that dialog, indeed.

Reply:
> this is going to take a while still anyway.

Seems like matej merged it 4 hours ago :D . I'm going to rebase and use it right now

Reply:
Done